bin/instant-workstation

```diff
@@ -94,7 +94,7 @@ vga_vendor=$( /usr/sbin/pciconf -l | /usr/bin/awk '/^vgapci/{ print substr($4,8,
 if test 8086 = "${vga_product}"
 then
 	case "x${fbsd_version}" in
-		"x12.0")
+		"x12.0"|"x12.1")
```

Comment:
Can we phrase this differently so that it covers future 12.2 as well?

Reply:
It looks like the 12.1 kmod is in the works (https://github.com/FreeBSDDesktop/kms-drm/commits/drm-v5.0-fbsd12.1) but hasn't been pulled into the packages tree yet, and I can't find mention of the 12.2 one yet. As far as I can tell, FreeBSD 12.2 uses the 12.0 kmod currently.

The best solution I can think of is to include `|"x12.2"` on that line and then shuffle things around once the respective releases happen. Any thoughts there?

Reply:
I was vaguely thinking `"x12."*` or something like that. I don't have the rules for case-matching at the front of my brain right now. And if there's a potential for differently-numbered kmods for different FreeBSD minor releases, let's leave it like this with things written-out explicitly.

Should probably warn when the FreeBSD version isn't recognized, though (eg. when 12.2 comes out and this script doesn't install a kmod).